drop-analytics/src/moose_impl.rs

```diff
@@ -110,17 +111,27 @@ impl MooseImpl {
         anyhow::ensure!(res.is_ok(), "Failed to initialize moose: {:?}", res.err());
 
         populate_context(&logger);
-        let res = moose::flush_changes();
-        moose_debug!(logger, res, "flush_changes");
-        res.context("Failed to flush moose context")?;
 
-        Ok(Self { logger })
+        let task_logger = logger.clone();
+        let context_update_task = tokio::spawn(async move {
+            let mut interval = tokio::time::interval(Duration::from_secs(60 * 10));
+            loop {
+                populate_context(&task_logger);
+                interval.tick().await;
+            }
+        });
+
+        Ok(Self {
+            logger,
+            context_update_task,
+        })
```

Comment:
I think that this should be a normal `std` thread.
Since moose is static and our tokio runtime is not, I see a lot of things that can go wrong here.
Just use plain old OS thread

Reply:
I'm unaware of the potential problems that this could cause, could you explain a bit more?

Reply:
So moose object has a static lifetime hence it lives until the libdrop so is unloaded (potentially without running its destructor).
But tokio runtime lives up until `norddrop_destory()` is called.
So I expect disasters to happen after `norddrop_destory()` is called, like panics and potential crashes

Reply:
Moreover, you simply don't need tokio here. It's a simple problem to solve, the OS thread suffices.
Tokio is meant for heavy IO-bound things like servers etc.

Reply:
Fair enough, changed it